src/custom/pages/Swap/ArrowWrapperLoader.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
```

Comment:
this shoudn't be in pages

please move it to custom/components as it's own file there